lib/auth/auth_with_roles.go

```diff
@@ -3078,6 +3078,25 @@ func (a *ServerWithRoles) GetSessionEvents(namespace string, sid session.ID, aft
 		return nil, trace.Wrap(err)
 	}
 
+	// emit a session recording view event for the audit log
+	user, err := a.GetCurrentUser(context.Background())
```

Comment:
It looks like the integration tests are failing on this line, because `a.GetCurrentUser` tries to check the user's roles, and can't find the role "Proxy". I'm not exactly sure why the integration test doesn't set up the Proxy system role, or why it's included in the site's list of roles.

A simpler option that would be to just use `a.context.User.GetName()` instead of calling `a.GetCurrentUser`, since you don't really need the role checking functionality just to emit the user name in the audit event.

Reply:
Awesome - thank you, I've changed it